_playbooks/playbook-pam.md

```diff
@@ -0,0 +1,367 @@
+
+This playbook is a collaboration between the General Services Administration (GSA) through the Federal CIO Council and the Department of Homeland Security (DHS) Cybersecurity and Infrastructure Security Agency (CISA)
+  
+Version 1.0  
+Last Updated: August xx, 2021
+
+# Executive Summary
+The Privileged User Playbook is a practical guide to **help federal agencies implement and manage a privileged user program**. Privileged users are a special user type that perform security-related duties. A privileged user program enables an agency to identify, track, monitor, and audit privileged users and accounts, effectively decreasing the cyber risk and impact on an agency's mission. **Unwanted behavior or compromise of privileged accounts are also responsible for the most high-profile federal security breaches**. It is a critical Identity, Credential, and Access Management capability to identify privileged users and access to high value assets. As part of agency insider threat programs, agencies can use this playbook to define privileged user program requirements and capabilities.
+
+There are three main use cases to identify a privileged account or user.
+1. Accounts used to manage IT infrastructure, resources of high value assets (HVA), and core systems (e.g. perform maintenance activities on human resource applications or databases).
+2. Help desk personnel with escalated privilege to perform security-relevant processes on endpoints (e.g. install software on user laptops or change endpoint configuration settings).
+3. Managers that approve access or accounts (e.g. a manager approving access requests to a risk reporting application).
+
+Privileged users are managed as a distinct and separate identity to decrease the risk and impact to agencies mission.
+1. These employees and contractors have the ability to jeopardize sensitive information or infrastructure, whether knowingly or unknowingly.
+2. Privileged users have the potential to compromise all three core elements of information security: availability, confidentiality, and integrity.
+
+Agencies can use this playbook to help plan and implement a logical privileged user program following government-wide best practices. This playbook includes a four step process aligned with the [Federal Identity, Credential, and Access Management (ICAM) Architecture](https://playbooks.idmanagement.gov/arch/){:target="_blank"}{:rel="noopener noreferrer"} and is designed for **insider threat, ICAM, and risk management professionals interested in identifying best practices for mitigation privileged user risk** for new or existing privileged user programs. Agencies are encouraged to tailor this playbook to fit their unique organizational structure, mission needs, and requirements. Other IT program participants, including cybersecurity program managers, may find value in incorporating this playbook approach in their planning.
```

Comment:
We originally called this the privileged access management playbook, but I changed it to privileged user playbook. For simplicity that this playbook talks about more than access and didn't want to use any vendor specific terms.

Reply:
Ok, But I don't think that privileged access management is a vendor specific term. It is a Cyber security industry standard term but not used as a specific product or vendor name.